### PR TITLE
feat: add construct orderbook example using websocket

### PR DESCRIPTION
**Pull request by 2hyunbin**
when i use a WebSocket to get the price, there was no properly documented example so i made it. For information on resolving the cross order book, i saw dydx docs (https://docs.dydx.exchange/api_integration-guides/how_to_uncross_orderbook).

### CODE REVIEW

**Comment by yogurtandjam** [comment]
thank you! some general thoughts:
- we've recently enforced using conventional commits for PR titles. i've updated this for you
- i'm okay with stamping as is, but would prefer to encapsulate the example logic a bit better. it's fairly complex and gets a bit hard to follow as a single flat function. 

if you feel like encapsulating will be more effort than it's worth, i'm happy to stamp as is. let me know what you think!

**Comment by 2hyunbin** [comment]
Thank you for applying the convention to the PR title. 
Your feedback that the code should be encapsulated to make it easier to read is reasonable. 
So I reflected the feedback. 
Please let me know if there is anything else that needs to be improved!

**Comment by 2hyunbin** [comment]
I have reflected your feedback, please check it @yogurtandjam
